### ISSUE TITLE
how to train on ucf101 from scratch?

### ISSUE THREAD

**Issue by xiadingZ** [mit-han-lab/temporal-shift-module#168]
Does anyone train tsm on ucf101  only using imagenet pretrained resnet50? No Kinetics pretrained.
I train tsm on ucf101 only using RGB stream, imagenet pretrained resnet50,  with hyper-parameters same as TSN, only get 83.5 accuracy.

**Comment by Fritskee** [comment]
Depending on the length of the clips it also helps with the amount of frames you sample. This can drastically improve (or decrease) the accuracy you obtain. In order to do so, set `--num_segments` to a higher value. Note that you might have to decrease `batch_size` (and the learning rate)  when you do this. 

As per your question in the title, "How to train from scratch" (with only imagenet), you can do this by not finetuning from an already provided checkpoint. Instead just start an entirely new training and use `--pretrain='imagenet'` which is also the default value for `--pretrain`.

**Comment by abdulhai1234** [comment]
> Does anyone train tsm on ucf101 only using imagenet pretrained resnet50? No Kinetics pretrained. I train tsm on ucf101 only using RGB stream, imagenet pretrained resnet50, with hyper-parameters same as TSN, only get 83.5 accuracy.

would you please help me to train the model on UCF101. I am facing problem while training it
